lib/trento.pm

```diff
@@ -726,16 +725,40 @@ sub trento_support {
     script_run(az_vm_ssh_cmd($cmd, $machine_ip));
     script_run("$scp_cmd:'*.tar.gz' $log_dir");
 
+    foreach my $this_file (split("\n", script_output('ls -1 ' . $log_dir . '*.tar.gz'))) {
+        upload_logs($this_file, failok => 1);
+    }
+    script_run("rm $log_dir *.tar.gz");
```

Comment:
I think this will fail as you cannot remove a directory with `rm`. It should be either:
```
script_run("rm $log_dir*.tar.gz");
```
... without the blank space, or:
```
script_run("rm $log_dir" . '*.tar.gz');
```
... which I don't like as it's almost the concatenation of 2 constants, or:
```
script_run('rm ' . $log_dir . '*.tar.gz');
```
... which I don't like as it's the concatenation of 2 constants and one scalar which can be placed between double quotes..

You can also probably do:
```
script_run("rm $log_dir/*.tar.gz");
```
... which IMO looks better.

Also, since return value it's not important, this `rm` can probably be a `rm -f` and this `script_run()` can be replaced with an `enter_cmd()`.

Reply:
Here an execution with the fixed `rm` http://openqaworker15.qa.suse.cz/tests/121956#step/cluster_deploy/95 (there's one extra slash that I can remove)